src/view/renderer.js

```diff
@@ -602,24 +602,27 @@ export default class Renderer {
 	 */
 	_updateDomSelection( domRoot ) {
 		const domSelection = domRoot.ownerDocument.defaultView.getSelection();
-		const oldViewSelection = domSelection && this.domConverter.domSelectionToView( domSelection );
 
-		if ( oldViewSelection && this.selection.isEqual( oldViewSelection ) ) {
-			return;
-		}
+		// Check whether dom selection needs fixing. If it does not need fixing maybe w don't have to update it at all.
```

Comment:
Actually, I don't get it. The comment says "If it does not need fixing maybe w don't have to update it at all." but the there is the condition which is executed only if the selection is correct.